sechub-wrapper-xray/src/test/java/com/mercedesbenz/sechub/wrapper/xray/util/IOHelperTest.java

```diff
@@ -0,0 +1,50 @@
+package com.mercedesbenz.sechub.wrapper.xray.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.mercedesbenz.sechub.wrapper.xray.XrayWrapperException;
+
+public class IOHelperTest {
+
+    final String testinputString = "test data";
```

Comment:
IMHO it would be better to define the content of "testInputString" inside the @beforeEach method.
Reasons:
- mainly because every default setup is done in @BeforeEach method, I did not see at the first glimpse, that there was a field declaration as well. To have defaults centrally defined is a good way.
- you could get rid of the "final" part
- you could change the test inputString inside a test 

Another option would be to change it to constant : 

```suggestion
    private final static String TEST_INPUT_STRING= "test data";
```
-> it would be clear that the data
- cannot be changed
- is constant
- initialized as field